tests/testutil.go

```diff
@@ -33,8 +33,23 @@ import (
 
 var once sync.Once
 
-// InitLogger initializes the logger for test.
-func InitLogger(cfg *tso.Config) (err error) {
+// InitSchedulingLogger initializes the logger for test.
+func InitSchedulingLogger(cfg *sc.Config) (err error) {
```

Comment:
Does resource manager server need similar function?

Reply:
We need further refactoring to merge them. For now, we don't use it.